### DISCUSSION TITLE
How to a text on the right side of a treeview item

### DISCUSSION THREAD

**Discussion by tonka3000** [microsoft/vscode-discussions#37] [Extension Development QnA] [answered]
Hi,

I try to add a text on the right side of a [vscode.TreeItem](https://code.visualstudio.com/api/references/vscode-api#TreeItem). I'm aware of the text label, the description and the how to add a command to a TreeItem via package.json.

But how can I add text on the right side like it is done in the *Timeline*?

![CleanShot 2022-06-12 at 15 20 21@2x](https://user-images.githubusercontent.com/3163807/173235411-3015db05-4729-4d69-a849-5d5eb97b2670.png)

**Comment by usernamehw** [comment, marked as answer]
Not exposed: https://github.com/microsoft/vscode/issues/107183
